app/views/content/blog/overcoming-challenges-to-become-a-teacher-hearing-impairment.md

```diff
@@ -39,3 +39,5 @@ The application process went smoothly thanks to the help of my [teacher training
 If someone came up to me and asked if they should teach, my answer would be yes and no! No, because teaching is not for everyone and it is a hugely demanding job so it is extremely important to understand the amount of patience, dedication and commitment you will need! However, teaching is a natural skill that anyone can learn as long as you are passionate about the subject and age group you’re teaching. Yes, because teaching allows students to absorb your knowledge and you can watch them flourish throughout the years, which makes the job so fascinating and unique!
 
 All of our advisers are experienced teachers who will provide you with additional support when preparing and applying for teacher training. [Sign up for a teacher training adviser](/teacher-training-advisers).
```

Comment:
@gemmadallmandfe now that this bit about advisers isn't in the indented bit at the bottom, it kinda feels like this is part of what the person writing it is saying (which it isn't - it's us). Maybe move this bit about advisers up to the top under the first para before the main part of the blog starts, where it talks about advisers?

Reply:
Missed this - good spot @EllieNodder !

I think maybe just combine the footer on both to say something like:

All of our advisers are experienced teachers who will provide you with additional support when preparing and applying for teacher training. Sign up for a teacher training adviser. You can also find out about the support you can get while training to teach if you’re disabled.

(reworded as necessary)